lambda/es-proxy-layer/lib/cfn.js

```diff
@@ -2,16 +2,18 @@ var Url=require('url')
 var Promise=require('bluebird')
 var cfnLambda=require('cfn-lambda')
 var request=require('./request')
+const qnabot = require("/opt/lib/logging")
```

Comment:
A simpler option is to introduce a module (e.g. qnabot-common) and require("qnabot-common/lib/logging") and include that module from node_modules instead of using `/opt`.

The lambda layer construction needs to  create the corresponding module under node_modules with the lib copied under it (needs updating the Makefile)